courses/free-courses-id.md

```diff
@@ -208,6 +208,7 @@
 * [Belajar JavaScript Async](https://www.youtube.com/playlist?list=PL-CtdCApEFH-I4CD6km3BcXqrhWAkY4et) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
 * [Dasar Pemrograman dengan JavaScriipt](https://www.youtube.com/playlist?list=PLFIM0718LjIWXagluzROrA-iBY9eeUt4w) - Web Programming UNPAS
 * [ExpressJS Tutorial Indonesia](https://www.youtube.com/playlist?list=PL9At9z2rvOC-sgzJx7rM_wMDONnEM4E0A) - Ekky Ridyanto (Balademy)
+* [JavaScript](https://devsaurus.com/javascript) - Devsaurus
```

Comment:
This link goes to one "class" in the devsaurus "course". you should make an entry for the complete course.

Reply:
change to javascript module course, done.